### PR TITLE
Add security policy to GraphQL-TSC.md

### PR DESCRIPTION
**Pull request by benjie**
As discussed in https://github.com/graphql/tsc/discussions/5 and https://github.com/graphql/graphql-wg/issues/825, this text was proposed by @brianwarner and has been lightly edited by myself.

According to [making changes to this document](https://github.com/graphql/graphql-wg/blob/main/GraphQL-TSC.md#making-changes-to-this-document):

> Pull requests against this document that do not conflict with the [Technical Charter](https://github.com/graphql/foundation/blob/master/GraphQL%20Specification%20Membership%20Agreement%20February%202019.pdf) can be merged provided the following conditions have been met:
>
> * There are no outstanding objections
> * There are two approvals by TSC members (not including the author)
> * The PR has been open for at least 72 hours

TL;DR: this PR can be merged after 2022-02-06T14:40:00Z assuming it gets two approvals and no objections

### CODE REVIEW

**Review by mjmahone — APPROVED** [review]

**Review by michaelstaib — APPROVED** [review]

**Review by leebyron — APPROVED** [review]

File: GraphQL-TSC.md [hunk 1]
```diff
@@ -286,6 +286,16 @@ If consensus cannot be reached, a pull request may still be landed after a vote
 
 An exception is made for errata or to update meeting logistics. These may be landed immediately, provided all EasyCLA checks have passed.
 
+### Security policy
+
+The GraphQL TSC is in an elevated position of trust within the GraphQL community. Security concerns that impact repos under the [`graphql` GitHub org](https://github.com/graphql/) (including reference implementations and official tools) may be responsibly disclosed to the TSC via [any current TSC member](https://github.com/graphql/graphql-wg/blob/main/GraphQL-TSC.md#tsc-members-1), with the expectation that they will be discussed and triaged by the TSC as a whole.
```
Comment (leebyron) [inline GraphQL-TSC.md:291]: @brianwarner should we set up an email alias to have a more clear disclosure guidance?

Comment (brianwarner) [reply GraphQL-TSC.md:291]: Sure thing. "security@graphql.org"? Maybe going to you and the operations@ alias?

In my experience these tend to attract a lot of spam and emails from bug bounty farms, so it probably makes sense going to someone who can triage before passing it along.

Comment (leebyron) [reply GraphQL-TSC.md:291]: This email is up and running if you want to swap this language @benjie

Comment (benjie) [reply GraphQL-TSC.md]: Done :+1:

**Comment by benjie** [comment]
I have one open for spec here: https://github.com/graphql/graphql-spec/pull/930 but it might make sense to create it in the https://github.com/graphql/.github repo instead so it automatically applies to all repos (but can still be overridden if desired).

What do you think @graphql/tsc?
